.github/workflows/integration-test.yml

```diff
@@ -0,0 +1,18 @@
+name: Integration
+on: [pull_request]
+
+jobs:
+  integration-test:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        tutor_version: ["<17.0.0", "<18.0.0"]
```

Comment:
Is there a specific reason we need to have a job for "<17.0.0" as well? Wouldn't it be enough to just use "<18.0.0"?

Reply:
> Is there a specific reason we need to have a job for "<17.0.0" as well? Wouldn't it be enough to just use "<18.0.0"?

Because only in Quince and Palm versions can you use the mount command